content/cli/v10/configuring-npm/package-json.mdx

```diff
@@ -903,7 +903,7 @@ The `devEngines` field aids engineers working on a codebase to all be using the
 
 You can specify a `devEngines` property in your `package.json` which will run before `install`, `ci`, and `run` commands.
 
-> Note: `engines` and `devEngines` differ in object shape. They also function very differently. `engines` is designed to alert the user when a dependency uses a differening npm or node version that the project it's being used in, whereas `devEngines` is used to alert people interacting with the source code of a project.
+> Note: `engines` and `devEngines` differ in object shape and functionality. `engines` is designed to alert the user when a dependency uses a different npm or node version than the project it's being used in, whereas `devEngines` is used to alert people interacting with the source code of a project.
```

Comment:
this isn’t quite accurate - `engines` is for a package to declare what npm and/or node versions it’s compatible with.

Reply:
Are you suggesting to get rid of the word `alert` or something else?

Reply:
```suggestion
> Note: `engines` and `devEngines` differ in object shape and functionality. `engines` is intended for a package to declare the node and/or npm versions it is compatible with (the default for both is `*`), whereas `devEngines` is used to declare the intended runtime (node, etc) and/or package manager (npm, etc) to be used by developers working directly in the source code of a project.
```

Reply:
Thanks! I think I see what you're saying now 👍 

What do you think of [e012ff6](https://github.com/npm/documentation/pull/1535/commits/e012ff639a34fd8d60b297e3fda9723e9f4f552c)?

Reply:
That's certainly correct! It would be nice to have the nuance that's contained in my suggestion, though :-)

Reply:
@styfle I like the nuance from @ljharb as well. If you want to update it with that I can see about getting this merged :)

Reply:
Sounds good. Please take a look at [ffbb248](https://github.com/npm/documentation/pull/1535/commits/ffbb2482cf412eb222136d903a7f804a088a2218)

Reply:
Looks good to me. The part about the default value is documented elsewhere, so it doesn't need to be here, but it seemed useful to reinforce.

Reply:
The reason I left out the default value is because this section of the docs is about `devEngines`, not `engines`.

We should keep the NOTE short(ish) or else move it out of the NOTe and into a normal paragraph.